src/utils/countdown.js

```diff
@@ -0,0 +1,30 @@
+const EventEmitter = require('events');
```

Comment:
Not sure, can you use import instead or require?
For example import { EventEmitter } from 'events';

Reply:
Yeah! fixed